doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -45,6 +45,10 @@ This subsection explains the solid objects (floating meshes) information. First
             # Z COR
             set z = 0
         end
+
+        subsection output
+            set output solid object = true
+        end
```

Comment:
Why did you to create a subsection? Could it not be just a free parameter?

Reply:
Yes it can be. It would be the only parameter not in a subsection. It's not a problem in itself. ]